src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+$month-length: "28" 28, "29" 29, "30" 30;
+$start-day: "mon" 1, "tue" 2, "wed" 3, "thu" 3, "fri" 5, "sat" 6, "sun" 7;
+$col-count: 7;
+$cell-size: 100px;
+$cell-gap: 1px;
+$size-calendar: ($col-count * $cell-size) + ($cell-gap  * 6);
+
+.calendar {
```

Comment:
add some inner spaces to avoid hiding a cell
![Screenshot 2023-04-20 at 20 24 17](https://user-images.githubusercontent.com/117587463/233442138-2e505b12-4ad7-4574-8fb5-bbfcda8272c4.png)